src/Files.App/Helpers/Environment/ElevationHelpers.cs

```diff
@@ -21,8 +21,8 @@ public static bool IsElevationRequired(string filePath)
 
 		public static bool IsAppRunAsAdmin()
 		{
-			using WindowsIdentity identity = WindowsIdentity.GetCurrent();
-			return new WindowsPrincipal(identity).IsInRole(WindowsBuiltInRole.Administrator);
+			WindowsIdentity identity = WindowsIdentity.GetCurrent();
```

Comment:
Is there no need for `using`?

Reply:
It works without it but I'm not sure if it makes a difference.

Reply:
Anyways, I think we need some more testing to see if this change unblocks users that have UAC turned off.

Reply:
I believe it is safer to leave the `using`. But I can't determine whether the modification on line 25 is correct or not.